build_tools/circle/build_wheel.sh

```diff
@@ -11,6 +11,8 @@ function build_wheel {
     ucs_tag="m"
     if [ "$ucs_setting" = "ucs4" ]; then
         ucs_tag="${ucs_tag}u"
+    elif [ "$pyver" = "3.8" ]; then
+        ucs_tag=""
     fi
```

Comment:
Can you make this:

```bash
    ucs_tag="m"
    if [ "$pyver" = "3.8" ]; then
        ucs_tag=""
    elif [ "$ucs_setting" = "ucs4" ]; then
        ucs_tag="${ucs_tag}u"
    fi
```

Just want to make sure we evaluate py version first, since `ucs_setting` and py version are not explicitly mutually exclusive.

Reply:
Done!